src/gluonnlp/utils/misc.py

```diff
@@ -651,3 +651,20 @@ def init_comm(backend, gpus):
             logging.info('GPU communication supported by KVStore')
 
     return store, num_workers, rank, local_rank, is_master_node, ctx_l
+
+
+def get_mxnet_available_ctx():
+    """Get the available contexts visible to MXNet
```

Comment:
this description doesn't describe what this function does accurately. you may want to clarify that if gpu is available, all available gpu contexts are returned, otherwise cpu context is returned.